client/src/components/home/Navbar.js

```diff
@@ -1,12 +1,30 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
 import styled from "styled-components";
+import { postData } from "../../helpers/fetch";
+function Navbar({ userData, setUserData }) {
+  const history = useHistory();
+  let linkTo;
+  const handleClick = () => {
+    !userData.message ? (linkTo = "/login") : (linkTo = "/logout");
+  };
 
-function Navbar() {
+  const handleFetch = async () => {
+    postData.headers.Authorization = `${userData.token}`;
```

Comment:
Could you help me with some context on the expected flow here: what do you hope to achieve with this request? i.e why not just direct the user to the cart page without this call?